### DISCUSSION TITLE
Self Hosted to Cloud Hosting

### DISCUSSION THREAD

**Discussion by alberk8** [Infisical/infisical#1201] [Q&A]
Is it possible to move a Self Hosted Infisical Instance to Paid Cloud Hosting by Infisical?

**Comment by dangtony98** [comment]
@alberk8 Currently it is not possible to do this easily; you'd have to move your secrets manually to Infisical Cloud.

We can put this on the roadmap tho — Feel free to open an [issue](https://github.com/Infisical/infisical/issues) for it!

**Reply by compgeniuses** [reply]
isnt it a loss for you guys not giving automated migration procedure?

**Reply by dangtony98** [reply]
This is on our roadmap but there are lot of other ongoing tasks at the moment, so we're tackling things one at a time 😄

**Comment by vmatsiiako** [comment]
Hi @alberk8! Yes, this is now possible and we'd be happy to look into this for you. Would you be able to reach out to team@infisical.com if you are still interested?
